crates/edr_provider/tests/eth_request_serialization.rs

```diff
@@ -3,10 +3,10 @@ mod common;
 use edr_eth::{
     filter::{LogFilterOptions, LogOutput, OneOrMore},
     transaction::EthTransactionRequest,
-    Address, BlockSpec, BlockTag, Bytes, PreEip1898BlockSpec, B256, U256, U64,
+    Address, BlockSpec, BlockTag, Bytes, PreEip1898BlockSpec, B256, U256,
 };
 use edr_evm::alloy_primitives::U160;
-use edr_provider::{IntervalConfigRequest, MethodInvocation, U64OrUsize};
```

Comment:
Is `U64OrUsize` still used anywhere else? If not, feel free to remove it.

Reply:
It's already removed in https://github.com/NomicFoundation/edr/pull/494/files#diff-f92267848751eff8250755669ad8dbb49e569a42f27bcdca2589fad3e50098a0L513

Reply:
My bad. Thanks!